File: pascal/strings/string_length.pas
(*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 *)

program StringLength;

uses
  strutils;

var
  word: string = 'hello';
  cite: string = 'Stat rosa pristina nomine, nomina nuda tenemus.';

begin
  writeln('word = ', word);
  writeln('length: ', length(word));
  writeln();

  writeln('cite = ', cite);
  writeln('length: ', length(cite));
  writeln('words quantity: ', wordcount(cite, [' ']));
  writeln();
end.

File: c++/strings/strings/string_length.cpp
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */
// SPDX-License-Identifier: GPL-3.0-or-later

#include <iostream>
#include <string>

int main()
{
  std::string word = "hello";
  std::cout << "word = " << word << "\n";
  std::cout << "length: " << word.length() << "\n\n";

  std::string cite = "Stat rosa pristina nomine, nomina nuda tenemus.";
  std::cout << "cite = " << cite << "\n";
  std::cout << "length: " << cite.length() << "\n\n";
}
